packages/fluentui/lerna.json

```diff
@@ -0,0 +1,5 @@
+{
+  "packages": ["**/*"],
+  "npmClient": "yarn",
+  "version": "0.45.0"
```

Comment:
We cannot use `useWorkspaces` there as Lerna has a validation for it: https://github.com/lerna/lerna/blob/6c4ee5294002c0e464f557b149516017eccf2a07/core/project/index.js#L74

However, it should not be an issue with publishing as Yarn Workspaces are not used there.